docs/source/dev/style.rst

```diff
@@ -3,148 +3,83 @@
 Coding Guidelines
 ==================
 
-.. attention::
-   The Coding Guidelines are currently revised
+Formatting
+----------
 
-General
--------
-
-* Use the ``.clang-format`` file supplied in alpaka's top-level directory to format your code. This will handle indentation,
-whitespace and braces automatically. Usage:
-
-.. code-block:: bash
-
-  clang-format-16 -i <sourcefile>
-
-* If you want to format the entire code base execute the following command from alpaka's top-level directory:
-
-.. code-block:: bash
-
-  find example include test -name '*.hpp' -o -name '*.cpp' | xargs clang-format-16 -i
-
-Windows users should use `Visual Studio's native clang-format integration
-<https://devblogs.microsoft.com/cppblog/clangformat-support-in-visual-studio-2017-15-7-preview-1/>`.
+Use the ``.clang-format`` file supplied in alpaka's top-level directory to format your code.
+This will handle indentation, whitespace and braces automatically.
+Checkout ``CONTRIBUTING.md`` on how to run it.
 
 Naming
 ------
 
 * Types are always in PascalCase (KernelExecCuda, BufT, ...) and singular.
-* Variables are always in camelCase (memBufHost, ...) and plural for collections and singular else.
+* Variables are always in camelCase (memBufHost, ...) and singlar by default. Use plural for collections.
```

Comment:
Mehmet mentioned that there is a typo `singlar` should be `singular`

Reply:
Fixed.